_rules/image-no-text-0va7u6.md

```diff
@@ -32,7 +32,13 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [rendered image resources][] in a [web page][].
+This rule applies to the [visible][] [image resource][] loaded on a [web page][]. Image resources can be loaded on a page through the following elements:
```

Comment:
1. I'm not sure what `[image resource]` is linking to. Is it the same as "image source" (in our current definition of "rendered image resources")?
2. I'm not a fan of the "can be" formulation because I think it is a bit ambiguous whether this is an exhaustive list or not (can it be loaded in a different way). If it is not exhaustive, then the text is ambiguous. If it is exhaustive, we should say so ("image resources are loaded when one of the following is true" or something similar).
3. The current definition of "rendered image resources" also has a notion of visibility of (pixels of) the image which makes sense (if the image is fully covered by some other image, who cares…)